website/docs/Queries.md

```diff
@@ -90,6 +91,16 @@ Usually query first argument can be a **string** or a **regex**. Some queries ac
 
 > getByText, getAllByText, queryByText, queryAllByText, findByText, findAllByText
 
+```ts
+getByText(
+  text: TextMatch,
+  options?: {
+    exact?: boolean;
+    normalizer?: (text: string) => string;
+  }
+): ReactTestInstance;
+```
+
 Returns a `ReactTestInstance` with matching text – may be a string or regular expression.
 
 This method will join `<Text>` siblings to find matches, similarly to [how React Native handles these components](https://reactnative.dev/docs/text#containers). This will allow for querying for strings that will be visually rendered together, but may be semantically separate React components.
```

Comment:
This is not related to this pr but isn't this false now ? If I'm correct byText queries do not match text rendered in several components

Reply:
I think you are right, we should revisit our `*ByText` handling in a separate issue/PR, as there are some things worth discussing, e.g. should we match text spread between different child `Text` instance, compat with Jest Native, compat with RTL behaviour